src/nplinker/config.py

```diff
@@ -27,10 +27,46 @@
 # Validataor parameter `required=False` means the setting (e.g. "loglevel") must not exist rather
 # than being optional. So don't set the parameter `required` if the key is optional.
 validators = [
-    Validator("loglevel", is_type_of=str),
-    Validator("logfile", is_type_of=str),
-    Validator("repro_file", is_type_of=str),
-    Validator("log_to_stdout", is_type_of=bool),
+    # General settings
```

Comment:
When will this config.py file be called within the call base? i.e., when will the validation happen?